gradle/libs.versions.toml

```diff
@@ -255,6 +255,7 @@ skikoLinuxArm64 = { module = "org.jetbrains.skiko:skiko-awt-runtime-linux-arm64"
 sqldelightAndroid = { module = "com.squareup.sqldelight:android-driver", version.ref = "sqldelight" }
 sqldelightCoroutinesExt = { module = "com.squareup.sqldelight:coroutines-extensions", version.ref = "sqldelight" }
 sqliteJdbc = { module = "org.xerial:sqlite-jdbc", version = "3.36.0" }
+statelyIsoCollections = { module = "co.touchlab:stately-iso-collections", version = "1.2.0" }
```

Comment:
From the [comments here](https://github.com/touchlab/Stately/tree/main/deprecated#stately-iso-collections), it seems like this library is completely deprecated and seems to say that the `ArrayDeque` that is now part of the Kotlin Standard Library should be enough. Do all the tests still pass if we swap to the stdlib version?

Reply:
>  […] it seems like this library is completely deprecated […]

The library isn't deprecated, but the module is. Unfortunately, the [proposed replacement (`stately-concurrent-collections`)](https://github.com/touchlab/Stately#stately-concurrent-collections) for this module [isn't stable yet](https://mvnrepository.com/artifact/co.touchlab/stately-concurrent-collections).

> […] and seems to say that the `ArrayDeque` that is now part of the Kotlin Standard Library should be enough.

A KMP version of `java.util.ArrayDeque` [was added in Kotlin 1.3.70](https://blog.jetbrains.com/kotlin/2020/03/kotlin-1-3-70-released/#Double-endedqueueimplementation:ArrayDeque). Like `java.util.ArrayDeque`, `kotlin.collections.ArrayDeque` isn't thread-safe.

> Do all the tests still pass if we swap to the stdlib version?

From the previous point, `ArrayQueue` isn't thread-safe. Swapping out `IsoArrayDeque` with `ArrayQueue` _could_ make all the tests pass, but those tests would presumably be a bit flaky.